src/main/java/com/minecolonies/coremod/blocks/BlockHutTownHall.java

```diff
@@ -22,4 +31,24 @@ public String getName()
     {
         return "blockHutTownHall";
     }
+
+    @Override
+    public void onBlockPlacedBy(
+            @NotNull final World worldIn, @NotNull final BlockPos pos, final IBlockState state, final EntityLivingBase placer, final ItemStack stack)
+    {
+        if(worldIn.isRemote)
+        {
+            return;
+        }
+
+        if(placer.getActiveHand().equals(EnumHand.MAIN_HAND))
+        {
+            Colony colony = ColonyManager.getClosestColony(worldIn, pos);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'colony' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)